app/src/androidTest/java/org/mozilla/fenix/syncintegration/gradlewbuild.py

```diff
@@ -21,20 +21,23 @@ def test(self, identifier):
 
         # Change path accordingly to go to root folder to run gradlew
         os.chdir('../../../../../../../..')
-        args = './gradlew ' + 'app:connectedGeckoNightlyDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=org.mozilla.fenix.syncintegration.SyncIntegrationTest#{}'.format(identifier)
+        cmd = './gradlew ' + 'app:connectedGeckoNightlyDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=org.mozilla.fenix.syncintegration.SyncIntegrationTest#{}'.format(identifier)
 
-        self.logger.info('Running: {}'.format(' '.join(args)))
+        self.logger.info('Running cmd: {}'.format(cmd))
 
+        out = ""
         try:
             out = subprocess.check_output(
-                args, shell=True)
+                cmd,
+                shell=True,
+                stderr=subprocess.STDOUT)
         except subprocess.CalledProcessError as e:
             out = e.output
             raise
         finally:
-            #Set the path correctly
+            # Set the path correctly
             testsPath = "app/src/androidTest/java/org/mozilla/fenix/syncintegration/"
             os.chdir(testsPath)
 
             with open(self.log, 'w') as f:
-                f.writelines(out)
+                f.write(out)
```

Comment:
The output is a single string, so using `write` is more efficient than using `writelines`.